packages/discussions/graphql/resolvers/Discussion/comments.js

```diff
@@ -245,6 +244,29 @@ module.exports = async (discussion, args, context, info) => {
     }
   }
 
+  /* 
+    AUTO = comments are sorted 
+    - by date if the first comment was created in the last 24 hours 
+    - or by votes otherwise 
+    the property resolvedOrderBy is just needed when DiscussionOrder === AUTO
+  */
+  let resolvedOrderBy
+  if (orderBy === 'AUTO') {
+    const sortedComments = [...comments].sort(
+      (a, b) => Date.parse(a.createdAt) - Date.parse(b.createdAt),
```

Comment:
When sorting arrays, we usually include [`d3-array`](https://github.com/orbiting/backends/blob/60c32fd1982f4bd312ba3b1eddd990a53d449173/packages/republik-crowdfundings/graphql/resolvers/Pledge.js#L2) which takes care of edge cases.

```suggestion
      (a, b) => ascending(a.createdAt, b.createdAt),
```

It's likely not to require `Date.parse` (albeit not tested).

If you're inclined to change this, make sure to import `d3-array` at top of file first.